astroquery/mast/cutouts.py

```diff
@@ -379,6 +425,13 @@ def get_cutouts(self, *, coordinates=None, size=5, sector=None,
 
         if moving_target:
 
+            # The Moving Targets service is currently only available for SPOC
+            if product.upper() != "SPOC":
+                product = "SPOC"
+                warnings.warn("Only SPOC is available for moving targets queries. Defaulting to SPOC.", InputWarning)
```

Comment:
ditto above, and since this code (and the docstring) is repeated, is there any way these could be refactored such that this checking only needs to be done once in the code?  That may be beyond the scope of this PR, but copy-pasted code & docs rings alarm bells for me that we could write a single function to handle this functionality instead of adding it to several functions.

Reply:
There are a few parts in `cutouts.py` with duplicate code so I'm in favor of leaving it how it is for now and opening up a new PR for this refactoring and addressing all of the duplicates that way